app/views/school_mailer/year2020_ects_added_confirmation.text.erb

```diff
@@ -0,0 +1,14 @@
+Hello,
+
+The following teachers from <%= @school.name %>'s 2020 cohort can now access their support materials:
+
+<% @participants.each do |participant| %>
+  - <%= participant[:full_name] %>
+<% end %>
+
+If you need to change any details, reply to this email.
+
+Thanks,
+
+Teacher Continuing Professional Development
+Department for Education
```

Comment:
Does it need any govuk styling? Or does notify does that for us?

Reply:
I'm pretty sure notify does it for us, looking at apply etc, there's no styling included in theirs

Reply:
You've got me wondering now, i'll double check..